### ISSUE TITLE
caniuse-css Fails to activate

### ISSUE THREAD

**Issue by ninthart** [myJSheart/caniuse-css#1]
Hi,

I installed this morning, as this looks like an excellent utility, but unfortunately it threw an error upon installation. The stack trace is below:

[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.14.0-beta3 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.2
**Thrown From**: [caniuse-css](https://github.com/myJSheart/caniuse-css) package 1.0.0

### Stack Trace

Failed to activate the caniuse-css package

```
At Cannot find module 'atom-message-panel'

Error: Cannot find module 'atom-message-panel'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (/Applications/Atom Beta.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._resolveFilename (/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (/app.asar/src/native-compile-cache.js:50:27)
    at /packages/caniuse-css/lib/caniuse-css.js:4:43)
    at Module._compile (/app.asar/src/native-compile-cache.js:109:30)
    at /app.asar/src/compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/app.asar/src/native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (/app.asar/src/package.js:796:27)
    at Package.module.exports.Package.activateNow (/app.asar/src/package.js:207:16)
    at /app.asar/src/package.js:876:25
    at Function.module.exports.Emitter.simpleDispatch (/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:240:20)
    at CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:3:59)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/app.asar/src/command-registry.js:160:19)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchApplicationMenuCommand (/app.asar/src/atom-environment.js:1158:28)
    at EventEmitter.outerCallback (/app.asar/src/application-delegate.js:320:25)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -3:51.8.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-twilight-updated-syntax.theme-one-dark-ui)
     -3:22.1.0 core:confirm (input.hidden-input)
     -0:02.5.0 caniuse-css:toggle (input.hidden-input)
```

### Non-Core Packages

```
atom-autocomplete-php 0.21.0 
atom-beautify 0.29.17 
atom-bootstrap4 1.4.0 
atom-commander 0.8.6 
atom-css-comb 3.1.0 
atom-fix-path 0.1.0 
atom-html-preview 0.1.22 
atom-image-resize 0.3.0 
atom-message-panel-service 0.0.1-1 
atom-minify 0.8.0 
atom-shortcuts 0.0.2 
atom-slack-snippets 0.2.0 
atom-ternjs 0.17.0 
atom-trello 0.7.1 
atom-zurb-foundation 2.0.0 
atomizr 0.23.0 
autocomplete-emojis 2.5.0 
autocomplete-html-entities 0.1.0 
autocomplete-modules 1.6.7 
autocomplete-paths 1.0.5 
autocomplete-project-paths 2.2.0 
autocomplete-wordpress-hooks 0.2.2 
autoprefixer 3.6.1 
browser-plus 0.0.87 
browser-refresh 0.9.0 
build 0.67.0 
build-grunt 0.4.0 
busy 0.7.0 
caniuse-css 1.0.0 
clone-cursor 1.1.2 
color-picker 2.2.5 
config-import-export 0.3.1 
csslint 1.1.5 
dash 1.7.0 
data-atom 0.24.1 
docblockr 0.9.1 
emmet 2.4.3 
emmet-snippets-compatibility 1.0.6 
expose 0.13.0 
file-icons 2.0.13 
filecolor 0.4.0 
flexbox-snippets 0.3.0 
git-control 0.8.2 
git-plus 7.2.0 
git-time-machine 1.5.4 
grunt-runner 0.14.0 
highlight-column 0.5.2 
highlight-selected 0.12.0 
html-entities 0.4.0 
imagemin 1.5.1 
improved-autocomplete-php 1.1.1 
language-jsont 0.2.6 
language-markdown 0.19.1 
language-postcss 1.3.0 
language-pug 0.0.19 
language-sql-mysql 0.3.1 
language-svg 0.9.2 
less-than-slash 0.16.0 
linter 1.11.21 
linter-csslint 1.3.3 
linter-jshint 3.0.2 
linter-jsonlint 1.3.0 
linter-markdown 3.1.0 
linter-php 1.3.1 
linter-phpmd 1.5.2 
linter-pug 1.3.0 
linter-scss-lint 3.0.4 
linter-stylelint 3.7.0 
linter-tidy 2.3.0 
livestyle-atom 0.2.7 
markdown-preview-enhanced 0.9.5 
markdown-preview-plus 2.4.1 
markdown-toolbar 0.1.2 
markdown-writer 2.6.3 
merge-conflicts 1.4.4 
minimap 4.26.8 
minimap-bookmarks 0.4.2 
minimap-codeglance 0.4.5 
minimap-cursorline 0.2.0 
minimap-find-and-replace 4.5.1 
minimap-highlight-selected 4.5.0 
minimap-linter 1.6.0 
minimap-pigments 0.2.2 
minimap-selection 4.4.0 
minimap-split-diff 0.3.2 
modular-keymaps 0.1.4 
modular-snippets 0.1.3 
monokai-php-css-syntax 0.2.2 
navigate 0.1.17 
open-in-browser 0.4.7 
php-cs-fixer 4.1.0 
pigments 0.39.0 
platformio-ide-terminal 2.2.2 
project-viewer 0.3.43 
quick-editor 0.4.3 
scratch 1.2.0 
script 3.14.1 
simple-drag-drop-text 0.3.1 
snippet-generator 0.3.2 
source-preview 0.5.0 
source-preview-pug 0.2.0 
source-preview-sass 0.1.6 
split-diff 1.1.1 
stylefmt 4.0.0 
svg-preview 0.11.0 
svgo 2.6.0 
sync-settings 0.8.1 
toggle-quotes 1.0.0 
tool-bar 1.0.1 
tool-bar-almighty 0.8.1 
transform 0.4.0 
tree-view-autoresize 1.5.0 
tree-view-copy-relative-path 1.0.1 
tree-view-git-branch 0.1.2 
tree-view-git-status 1.3.0 
tree-view-open-files 0.3.0 
twilight-updated-syntax 0.3.0 
twilightesque-syntax 0.1.0 
wordcount 2.10.3 
wordpress-api 1.2.2 
wordpress-dictionary 4.4.0 
xeta-syntax 0.5.0 
```

Basically, it asks for the _atom-message-panel_ module to be installed. That package has now been depreciated in favour of a new package called _atom-message-panel-service_. I tried installing that package, but it didn't help.

Thanks :)

**Comment by myJSheart** [comment]
@ninthart Thank you for telling me this problem. I used  ```atom-message-panel``` to display results and abandon when I release. I will fix it ASAP.

**Comment by myJSheart** [comment]
@ninthart I update this plugin to 1.1.3 and remove atom-message-panel. Please try to find out whether this bug has been fixed

**Comment by ninthart** [comment]
@myJSheart Yep, it's been fixed. Many thanks for this, it's working beautifully 👍
